src/comet_llm/experiment_api/request_exception_wrapper.py

```diff
@@ -62,3 +67,8 @@ def _is_on_prem(url: str) -> bool:
     parsed = urllib.parse.urlparse(url)
     root = f"{parsed.scheme}://{parsed.hostname}/"
     return root != "https://www.comet.com/"
+
+
+def _handle_response(response: requests.Response) -> Optional[str]:
+    sdk_error_code = json.loads(response.text)["sdk_error_code"]
+    return logging_messages.SDK_ERROR_CODES_LOGGING_MESSAGE.get(sdk_error_code)
```

Comment:
Move handle_response to a separate module and make it a public function. Move the dictionary with SDK error codes to this module as well.

Upd. maybe `failed_response_handler.handle(...)` or something like that is a better name for such module and function since it's only for failed responses :)